docs/content.zh/docs/deployment/overview.md

```diff
@@ -205,6 +205,10 @@ non-blocking, will lead to the "next" job starting before "this" job finishes.
 The Application Mode allows for multi-`execute()` applications but 
 High-Availability is not supported in these cases. High-Availability in Application Mode is only
 supported for single-`execute()` applications.
+
+Additionaly, when any of multiple running jobs in Application Mode (submitted for example using 
+`executeAsync()`) gets cancelled, all jobs will be stopped and the JobManager shuts down. 
+Finishing (by the sources shutting down) of any of multiple running jobs is supported.
```

Comment:
I find this a bit unclear, since finishing a job is not an active action triggered by the user. What do you think of something along the lines of "Regular job completions (by the sources shutting down) are supported."?